.github/workflows/nightly-release.yml

```diff
@@ -45,31 +45,30 @@ jobs:
         run: make build
       - name: Create release artifact
         run: |
-          zip -r kakarot-nightly-build.zip build/
-      - name: Generate release notes
+          zip -r kakarot-pre-release-build.zip build/
+      - name: Generate and process release notes
+        id: release_notes
         run: |
-          echo "Nightly build created on $(date +'%Y-%m-%d %H:%M:%S')" > release_notes.md
-          echo "Commit: ${{ github.sha }}" >> release_notes.md
-          echo "Recent changes:" >> release_notes.md
-          git log -n 5 --pretty=format:"- %s" >> release_notes.md
-      - name: Create GitHub release
-        id: create_release
-        uses: actions/create-release@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          tag_name: nightly-${{ github.sha }}
-          release_name: Nightly Release ${{ github.sha }}
-          body_path: release_notes.md
-          draft: false
-          prerelease: true
+          {
+            echo "Pre-release build created on $(date +'%Y-%m-%d %H:%M:%S')"
+            echo "Commit: ${{ github.sha }}"
+            echo "Recent changes:"
+            git log -n 5 --pretty=format:"- %s"
+          } > release_notes.md
 
-      - name: Upload release asset
-        uses: actions/upload-release-asset@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          RELEASE_BODY=$(cat release_notes.md)
+          RELEASE_BODY="${RELEASE_BODY//'%'/'%25'}"
+          RELEASE_BODY="${RELEASE_BODY//$'\n'/'%0A'}"
+          RELEASE_BODY="${RELEASE_BODY//$'\r'/'%0D'}"
+          echo "RELEASE_BODY=$RELEASE_BODY" >> $GITHUB_OUTPUT
+      - name: Create Pre-Release and Upload Asset
+        uses: svenstaro/upload-release-action@v2
```

Comment:
From what I saw `make_latest: Mark the release as the latest release for the repository (Default: true).`
We might want to make it to false in case someone wants to point to the latest stable release (and not a pre one)

Reply:
ah I thought latest and pre-release were mutually exclusive but it appears not, ty